client/src/components/DatePicker/DatePicker.css

```diff
@@ -1,25 +1,23 @@
 @font-face {
-  font-family: Raleway;
-  src: url("https://fonts.googleapis.com/css?family=Raleway&display=swap");
-}
-
-.date-picker {
-  display: block;
-  font-family: Raleway;
-  font-style: normal;
-  font-weight: 800;
-  font-size: 18px;
-  line-height: 21px;
-}
-
-.requested-date-input {
-  box-shadow: 5px 10px;
-  font-family: Raleway;
-  font-style: normal;
-  font-weight: normal;
-  font-size: 18px;
-  line-height: 21px;
-  text-align: center;
-  margin: 0.4rem 0;
-  background: rgba(68, 31, 14, 0.2);
-}
+    font-family: Raleway;
+    src: url("https://fonts.googleapis.com/css?family=Raleway&display=swap");
+  }
+  .date-picker {
+    display: block;
+    font-family: Raleway;
+    font-style: normal;
```

Comment:
Can you remove the date picker files from your PR?

Reply:
Don't delete the files! Just remove them from your PR

Reply:
removed